python/pyspark/context.py

```diff
@@ -952,6 +952,11 @@ def dump_profiles(self, path):
         """
         self.profiler_collector.dump_profiles(path)
 
+    def getConf(self):
```

Comment:
Ur, did what I missed to do, e.g. comments?
Scala/Java has already that and we need to use that for singleton pattern in streaming word count example.